OrbitService/ServiceUtils.cpp

```diff
@@ -75,6 +75,9 @@ ErrorMessageOr<std::vector<ModuleInfo>> ParseMaps(std::string_view proc_maps_dat
 
     const std::string& module_path = tokens[5];
 
+    // This excludes mapped character or block devices.
+    if (module_path.rfind("/dev/", 0) == 0) continue;
```

Comment:
I wasn't aware of this function rfind. I assume this usage is basically just checking if the prefix matches? 

I dont really care, but there is also `bool absl::StartsWith(string, prefix)`. Maybe its easier to read

Reply:
+1